packages/css/src/suggestion.css

```diff
@@ -0,0 +1,113 @@
+.ds-suggestion {
+  --dsc-suggestion-option-background--selected: var(--ds-color-background-subtle);
+  --dsc-suggestion-option-border-color: var(--ds-color-base-default);
+  --dsc-suggestion-clear-gap: var(--ds-spacing-2);
+  --dsc-suggestion-clear-padding: var(--ds-sizing-1);
+  --dsc-suggestion-clear-size: var(--ds-sizing-9);
+  --dsc-suggestion-clear-icon-url: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24'%3E%3Cpath fill='currentColor' d='M6.53 5.47a.75.75 0 0 0-1.06 1.06L10.94 12l-5.47 5.47a.75.75 0 1 0 1.06 1.06L12 13.06l5.47 5.47a.75.75 0 1 0 1.06-1.06L13.06 12l5.47-5.47a.75.75 0 0 0-1.06-1.06L12 10.94z'/%3E%3C/svg%3E");
+  --dsc-suggestion-option-checkmark-url: url('data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%221em%22%20height%3D%221em%22%20fill%3D%22none%22%20viewBox%3D%220%200%2024%2024%22%20focusable%3D%22false%22%20role%3D%22img%22%3E%3Cpath%20fill%3D%22currentColor%22%20fill-rule%3D%22evenodd%22%20d%3D%22M18.998%206.94a.75.75%200%200%201%20.063%201.058l-8%209a.75.75%200%200%201-1.091.032l-5-5a.75.75%200%201%201%201.06-1.06l4.438%204.437%207.471-8.405A.75.75%200%200%201%2019%206.939%22%20clip-rule%3D%22evenodd%22%3E%3C%2Fpath%3E%3C%2Fsvg%3E');
+  --dsc-suggestion-option-checkmark-size: var(--ds-sizing-7);
+  --dsc-suggestion-list-gap: var(--ds-spacing-2);
```

Comment:
We forget to rename the sizing/spacing variables to new naming. 

<img width="323" alt="image" src="https://github.com/user-attachments/assets/734095ed-c85d-4348-8b0d-bc7b6a8b913b" />

Reply:
will fix